packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
 export interface ITooltipProps
   extends Partial<ThemeProps<DefaultTheme>>,
     React.HTMLAttributes<HTMLDivElement> {
-  /** Appends the tooltip to the body element */
+  /** Appends the tooltip to the element provided */
   appendToNode?: Element;
+  /** Adds an arrow to the tooltip */
   hasArrow?: boolean;
-  /** Milliseconds of delay before open/close of tooltip is initiated */
+  /** Adds milliseconds of delay to the opening and closing of the tooltip */
   delayMS?: number;
-  /** Whether Popper should update based on DOM resize events */
+  /** Allows the tooltip to reposition during browser resize events */
   eventsEnabled?: boolean;
+  /** Sets the ID of the tooltip */
   id?: string;
+  /** Defines the content of the tooltip */
   content: React.ReactNode;
   /**
-   * These placements differ from the default naming of Popper placements to help
+   * Adjusts the placement of the tooltip. Placement values differ from Popper placement values to
    * assist with RTL layouts.
```

Comment:
Again, I think the simplicity of your previous description is ideal:

https://github.com/zendeskgarden/react-components/pull/957/files#diff-8fc5bdeb0d77c73a3b03ce92de8df5ff9a393a3b092cc638a7700df6c8f42ec3L30-L33